test/screenshot.test.js

```diff
@@ -10,7 +10,8 @@ const { getURLsFromSitemap } = require('./support/sitemap');
 const { writeFile, mkdir } = fsPromises;
 
 const LOCAL_HOST = `http://localhost:${process.env.JEST_PORT}`;
-const REMOTE_HOST = 'https://design.login.gov';
+const REMOTE_HOST =
+  'https://federalist-340d8801-aa16-4df5-ad22-b1e3e731098e.sites.pages.cloud.gov/site/18f/identity-design-system';
```

Comment:
WDYT of trying to have this do a `git checkout main` or something instead of continuing to rely on federalist? My expectation is that by shutting down `design.login.gov` we'd remove everything published?

Reply:
For the near term, I'm expecting we'll continue to use Federalist to be able to use the branch preview functionality, even if the main https://design.login.gov site will start to show a 404 (which, per https://github.com/18F/identity-design-system/pull/333#issuecomment-1528093907, is still going to be hosted by Federalist).

Reply:
sounds good!